source/StarField.cpp

```diff
@@ -127,13 +127,20 @@ void StarField::SetHaze(const Sprite *sprite, bool allowAnimation)
 void StarField::Draw(const Point &pos, const Point &vel, double zoom) const
 {
 	double baseZoom = zoom;
+	int j = 1;
+
+	if(Preferences::Has("Parallax background"))
+		j = 4;
 
 	// Draw the starfield unless it is disabled in the preferences.
 	if(Preferences::Has("Draw starfield"))
+	{
+	for(float i = 0; i < j; i++)
```

Comment:
```suggestion
	for(float i = 0; i < passes; i++)
```
i could have a better name too, and it doesnt have to be `passes` for j just not `j`